api/apiclient.go

```diff
@@ -498,7 +527,19 @@ func dialAPI(ctx context.Context, info *Info, opts0 DialOpts) (*dialResult, erro
 		return nil, errors.New("no API addresses to connect to")
 	}
 
-	addrs := info.Addrs[:]
+	addrs := make([]*url.URL, 0, len(info.Addrs))
+	for _, addr := range info.Addrs {
+		url, ok := parseURLWithOptionalScheme(addr)
```

Comment:
When would addr have or not have the scheme? To date, info.Addrs has just been `host:port` I think? WHich means we'd always be adding the default "wss". What flow would result in the scheme being part of addr?